src/main/webapp/js/countryCodes.es6

```diff
@@ -1,10 +1,10 @@
-'use strict';
-
 // Country code data from:
 // https://gist.github.com/maephisto/9228207
 // https://github.com/michaelwittig/node-i18n-iso-countries/blob/master/codes.js
 
-var countryToCode = {
+/* eslint quote-props: ["error", "always"] */
```

Comment:
To make sure that ESLint doesn't complain about the properties being in quotes. Otherwise it's really ugly.